From cc8c56510114a81357ce73392ec0d416e4c86d9a Mon Sep 17 00:00:00 2001
From: Bruno Haible <bruno@clisp.org>
Date: Sat, 12 Feb 2022 21:26:57 +0100
Subject: [PATCH 3/3] loongarch: Fix gcc installation.

* cross-tools/cross.conf (loongarch64): Change gcc version to match what
it actually reports.
---
 ChangeLog              | 12 ++++++++++++
 cross-tools/cross.conf |  2 +-
 2 files changed, 13 insertions(+), 1 deletion(-)

diff --git a/ChangeLog b/ChangeLog
index 1b51050..f932b24 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,3 +1,15 @@
+2022-02-12  Bruno Haible  <bruno@clisp.org>
+
+	loongarch: Fix gcc installation.
+	* cross-tools/cross.conf (loongarch64): Change gcc version to match what
+	it actually reports.
+
+2022-02-12  Bruno Haible  <bruno@clisp.org>
+
+	Fix cross-tools build.
+	* cross-tools/cross-build.sh (func_build_gcc): Set LD_LIBRARY_PATH, so
+	that libisl.so.15 gets found during installation.
+
 2022-02-12  Bruno Haible  <bruno@clisp.org>
 
 	loongarch: Allow for continuous integration.
diff --git a/cross-tools/cross.conf b/cross-tools/cross.conf
index b6eccde..7ed71a4 100644
--- a/cross-tools/cross.conf
+++ b/cross-tools/cross.conf
@@ -24,4 +24,4 @@ s390         s390-linux              3.1          2.12.90.0.7  s390-linux
 s390x        s390x-linux             5.4.0        2.24         s390x-linux
 riscv32      riscv32-linux           7.3.0        2.30         riscv32-linux
 riscv64      riscv64-linux           7.3.0        2.30         riscv64-linux
-loongarch64  loongarch64-linux       12.0.0-loong 2.38         loongarch64-linux
+loongarch64  loongarch64-linux       12.0.1-loong 2.38         loongarch64-linux
-- 
2.20.1

